Corona-Warn-App/src/main/res/layout/contact_diary_day_fragment.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:contentDescription="@string/settings_title"
+        android:focusable="true">
+
+        <include
```

Comment:
Can we evaluate whether we need to use the layout include? Maybe we can just use the `ToolBar` class from Android.

Reply:
Hmm i went with the the way it is done in the rest of the app for consistency but i'm sure this can be discussed